stl/endpoint/pdp.py

```diff
@@ -318,7 +318,7 @@ def __get_price_key(pricing) -> str:
     def __get_price_rate(pricing) -> int | None:
         if pricing:
             price_key = Pdp.__get_price_key(pricing)
-            return int(pricing['structuredStayDisplayPrice']['primaryLine'][price_key].lstrip('$').replace(',', ''))
+            return int(pricing['structuredStayDisplayPrice']['primaryLine'][price_key].lstrip('$').replace(',', '').replace('\xa0USD', ''))
```

Comment:
We can use `unicodedata.normalize`

```suggestion
            return int(normalize('NFKD', pricing['structuredStayDisplayPrice']['primaryLine'][price_key]).lstrip('$').replace(',', ''))
```

At the top of the file, we'd also need to add:
```python
from unicodedata import normalize
```